gatsby/src/docs/workflow/releases/health.mdx

```diff
@@ -86,6 +86,13 @@ To benefit from the health data you must use at least version 1.4.0 of the React
 
 For more details, see the [full documentation on using Release Health with React Native](/platforms/react-native/#release-health).
 
+### Native (C/C++)
+
+To benefit from the health data you must use at least version 0.4.0 of the Native SDK.
```

Comment:
suggest: To benefit from health data, you must use at least 0.4.0 of our Native SDK. Metrics collection for release health is, by default, enabled, unless you specifically disabled collecting in the initialization options for the SDK.